### PR TITLE
Update `update_bundled_ca_certificates` utility script for directory nesting

### PR DESCRIPTION
**Pull request by RochesterinNYC**
- Also updates the ssl certs with the minimum certs needed and nests them in directories named for the hosts they can authenticate against. The arrangement of the certs changed/occurred because the ssl certs in Rubygems were modeled after the ssl cert arrangement in Bundler. However, this arrangement was produced through manual testing and configured/arranged in bundler/bundler#4380 instead of programmatically. I'm unsure how I would go about writing or updating the `update_bundled_ca_certificates` script in a way that could reproduce this current manual arrangement. Hence, this PR includes a commit for rearranging of the ssl certs based off the programmatic utility script (instead of a manual process).
- Related to discussion at #1555

cc: @segiddins @indirect

### CODE REVIEW

**Comment by RochesterinNYC** [comment]
Fixed this PR via updating the `Manifest.txt`.

**Comment by segiddins** [comment]
@dwradcliffe it'd be awesome if you could also have a look at this

File: util/update_bundled_ca_certificates.rb [hunk 1]
```diff
@@ -79,13 +86,27 @@ def test_uri uri, certificates
   nil
 end
 
+def hostname_certificate_mapping(certificates)
```
Comment (dwradcliffe) [inline util/update_bundled_ca_certificates.rb]: This should match the style used in the file already (no parentheses)

**Comment by dwradcliffe** [comment]
Functionality LGTM and tested locally

**Comment by RochesterinNYC** [comment]
Thanks @dwradcliffe. Fixed the parentheses and updated the PR.

**Comment by segiddins** [comment]
@homu r+

File: util/update_bundled_ca_certificates.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
```
Comment (eestrella198) [inline util/update_bundled_ca_certificates.rb]: L

**Pull request merged**